src/test/developTest/resources/wdr/wdr_sendung_day.html

```diff
@@ -0,0 +1,1334 @@
+</optgroup>
+</select>
+<input value="Zum Ressort wechseln" type="submit">
+</form>
+</div>
+</noscript>
+<div class="masternavi" role="navigation" data-ctrl-navigation="{'id':'clayer','action':{'default':['closeEntries','closeEntriesOnVanish']}, 'refreshOnAdded':'true'}">
+<ul>
+<li class="active mt" >
+<a href="/mediathek/index.html">
+Mediathek
+</a>
+</li>
+<li class="tv" data-ctrl-clayer-entry="{'id':'tvlayer'}">
+<span class="subressort collapsed layer" data-ctrl-clayer-tvlayer-trigger="{'action':{'default':['closeOtherNavis','toggleBody','openLayerEntry'],'s':['preventToggle'],'xs':['preventToggle'] }}">
+<a href="/fernsehen/index.html" data-ctrl-attributeswap="{'action':{'xs': {'href': '/fernsehen/hp-fernsehlayer-100.html'}, 's': {'href': '/fernsehen/hp-fernsehlayer-100.html'}}}">Fernsehen</a>
+</span>
+<div class="subressorts" data-ctrl-clayer-tvlayer-body="{}" id="tvlayer">
+<div class="section sectionA index rubric" data-ctrl-loadable="{'id':'lytv','action':{'s':['disable'],'xs':['disable']}}">
+<div class="con" data-ctrl-lytv-target="{}"></div>
+<span data-ctrl-lytv-source="{'action':{'default':['load']},'url':'/fernsehen/hp-fernsehlayer-100~fragment.html','method':'GET','singleLoad':false}">
+<a class="layerTrigger" href="#"></a>
+</span>
+</div>
+</div>
+</li>
+<li class="radio" data-ctrl-clayer-entry="{'id':'radiolayer'}">
+<span class="subressort collapsed layer" data-ctrl-clayer-radiolayer-trigger="{'action':{'default':['closeOtherNavis','toggleBody','openLayerEntry'],'s':['preventToggle'],'xs':['preventToggle'] }}">
+<a href="/radio/index.html" data-ctrl-attributeswap="{'action':{'xs': {'href': '/radio/index.html'}, 's': {'href': '/radio/index.html'}}}">Radio</a>
+</span>
+<div class="subressorts" data-ctrl-clayer-radiolayer-body="{}" id="radiolayer">
+<div class="section sectionA index rubric" data-ctrl-loadable="{'id':'lyradio','action':{'s':['disable'],'xs':['disable']}}">
+<div class="con" data-ctrl-lyradio-target="{}"></div>
+<span data-ctrl-lyradio-source="{'action':{'default':['load']},'url':'/radio/radiolayer-100~fragment.html','method':'GET','singleLoad':false}">
+<a class="layerTrigger" href="#"></a>
+</span>
+</div>
+</div>
+</li>
+</ul>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+</div>
+<div id="content" class="home" data-page-url="http://www1.wdr.de/mediathek/video/sendungverpasst/index.html">
+<div class="wrapper" role="main" data-ctrl-layoutable="{'id':'sectionlayout','action':{'default':['equalHeight'],'xs':['disable'],'s':['disable']},'timeToWait':'50','refreshOnAdded':'true','offsetY':'0'}">
+<div class="section sectionA hidden" >
+<div class="breadcrumb screenReader">
+<p class="hidden">Sie befinden sich hier: </p>
+<ul>
+<li>
+<a href="/index.html">WDR</a>
+</li>
+<li>
+<a href="/mediathek/index.html">Mediathek</a>
+</li>
+<li>
+<a href="/mediathek/video/index.html">Video</a>
+</li>
+<li>
+Sendung verpasst
+</li>
+</ul>
+</div>
+</div>
+<a name="goToContent" class="hidden"></a>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "name" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AS1827)